modules/ocp-running-rsync-root-or-non-root.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// migration_toolkit_for_containers/mtc-direct-migration-requirements.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ocp-running-rsync-root-or-non-root_{context}"]
+= Running Rsync as either root or non-root
+
+{OCP} environments have the `PodSecurityAdmission` controller enabled by default. This controller requires cluster administrators to enforce Pod Security Standards by means of namespace labels. All workloads in the cluster are expected to run one of the following Pod Security Standard levels: `Privileged`, `Baseline` or `Restricted`. Every cluster has its own default policy set.
+
+To guarantee successful data transfer in all environments, {mtc-full} ({mtc-short}) 1.7.5 introduced changes in Rsync pods, including running Rsync pods as non-root user by default. This ensures that data transfer is possible even for workloads that do not necessarily require higher privileges. This change was made because it is best to run workloads with the lowest level of privileges possible.
+
+
```

Comment:
Each heading should have an anchor ID to possibly link to, here's one suggestion:
```suggestion
[id="manually-overriding-default-nonroot-operation_{context}"]
```

Feel free to choose something else so long as it has the context at the end.